content/src/fieldConfig/profile-industries.json

```diff
@@ -79,76 +91,70 @@
+        "radioButtonAnnualText": "Annual",
+        "radioButtonConditionalText": "Conditional"
       },
       "certifiedInteriorDesigner": {
         "radioButtonYesText": "Yes",
         "radioButtonNoText": "No",
         "header": "### Certified Interior Designer",
         "description": "WIll you market as a certified interior designer?"
       },
-      "realEstateAppraisalManagement": {
+      "cpa": {
         "radioButtonYesText": "Yes",
         "radioButtonNoText": "No",
-        "header": "### Appraisal Management",
-        "description": "Are you looking to start an appraisal management company?"
-      }
-    },
-    "OWNING": {
-      "cannabisLicense": {
-        "description": "",
-        "header": "",
-        "radioButtonAnnualText": "",
-        "radioButtonConditionalText": ""
-      },
-      "cpa": {
-        "header": "",
-        "description": "",
-        "radioButtonYesText": "",
-        "radioButtonNoText": ""
+        "header": "### Public Accounting Services",
+        "description": "Will you provide Certified Public Accounting Services?"
       },
       "employmentAgency": {
         "header": "",
-        "description": ""
+        "description": "Learn about other `employment and personnel industries|employment-agency-types`."
+      },
+      "homeBased": {
+        "radioButtonYesText": "Yes",
+        "radioButtonNoText": "No",
+        "header": "### Home-based business",
+        "description": "Do you plan to be a `home-based business|home-based-business`?"
       },
       "homeContractor": {
         "header": "",
-        "description": ""
+        "description": "`Learn what activities qualify as a Home Improvement Contractor|home-contractors-activities`"
       },
       "liquorLicense": {
-        "header": "",
-        "description": "",
-        "radioButtonYesText": "",
-        "radioButtonNoText": ""
-      },
-      "staffingService": {
-        "radioButtonYesText": "",
-        "radioButtonNoText": "",
-        "header": "",
-        "description": ""
-      },
-      "certifiedInteriorDesigner": {
-        "radioButtonYesText": "",
-        "radioButtonNoText": "",
-        "header": "",
-        "description": ""
+        "radioButtonYesText": "Yes",
+        "radioButtonNoText": "No",
+        "header": "### Liquor",
+        "description": "Do you plan to serve liquor, including beer or wine?"
       },
       "realEstateAppraisalManagement": {
-        "radioButtonYesText": "",
-        "radioButtonNoText": "",
-        "header": "",
-        "description": ""
+        "radioButtonYesText": "Yes",
+        "radioButtonNoText": "No",
+        "header": "### Appraisal Management",
+        "description": "Are you looking to start an appraisal management company?"
+      },
+      "staffingService": {
+        "radioButtonYesText": "Yes",
+        "radioButtonNoText": "No",
+        "header": "### Staffing Services",
+        "description": "Will you provide staffing services?"
```

Comment:
The industry specific questions should all be blank for owning, since they don't select an industry

Reply:
@dsueltenfuss When Poppy 'graduates' and navigates to the profile from the dashboard, their business persona is changed to "Owning". Content should be required otherwise the component loads without any content.

Reply:
We should discuss this further.

Reply:
Will remove.